testing/pypy/APKBUILD

```diff
@@ -0,0 +1,49 @@
+# Contributor:Mihnea Saracin <mihnea.saracin@rinftech.com>
+# Maintainer:Mihnea Saracin <mihnea.saracin@rinftech.com>
+pkgname=pypy
+pkgver=5.7.1
+pkgrel=0
+pkgdesc="pypy2"
+url="https://bitbucket.org/pypy"
+arch="all"
+license="MIT"
+depends="libssl1.0 libcrypto1.0 libffi libbz2 "
+makedepends="gcc make python linux-headers  libffi-dev pkgconf zlib-dev bzip2-dev 
```

Comment:
At least `gcc`, `make`, `libc-dev` and `tk` can likely be removed from `makedepends`: The former three belong to `build-base`, and the latter *should* be implied by `tk-dev`. Also, does `pypy` not build against LibreSSL? I would suggest `libressl-dev` if it does, because we try to limit the number of packages linking against OpenSSL.